src/priceClient/adapters/across.ts

```diff
@@ -0,0 +1,50 @@
+import axios, { AxiosError } from "axios";
+import get from "lodash.get";
+import { Logger, msToS, PriceFeedAdapter, TokenPrice } from "../priceClient";
+
+type AcrossPrice = { price: number };
+
+export class PriceFeed implements PriceFeedAdapter {
+  public readonly platforms = ["ethereum"];
+
+  constructor(public readonly logger: Logger, public readonly name: string, public readonly host?: string) {
+    // Allow host to be overridden for test or alternative deployments.
+    if (this.host === undefined) this.host = "across.to";
+  }
+
+  async getTokenPrice(address: string, currency: string, platform: string): Promise<TokenPrice> {
+    if (!this.platforms.includes(platform)) throw Error(`Platform ${platform} not supported.`);
+
+    // Note: Assume price is 60 seconds old since the API does not provide a timestamp.
+    const now = msToS(Date.now()) - 60;
+    const acrossPrice: AcrossPrice = await this.query(address, currency);
+
+    return { address: address, price: acrossPrice.price, timestamp: now };
+  }
+
+  // todo: Support bundled prices in the API endpoint.
```

Comment:
Need to highlight this comment. If we don't implement bundling in the Across API endpoint then we'll probably end up issuing _n_ requests on each price lookup because the PriceClient assumes that the configured PriceFeed supports bundling.

Fortunately, given that we're dogfooding so much here it should be achievable to support bundling via the existing Coingecko implementation, or alternatively via the PriceClient itself (inception...).